CMakeLists.txt

```diff
@@ -849,6 +856,70 @@ ELSEIF ("${CMAKE_SYSTEM_NAME}" MATCHES "SunOS")
     ENDIF (LIBUV_FOUND)
 ENDIF ()
 
+# h2olog
+# If BCC is installed to a non-default directory,
+# set something like `PKG_CONFIG_PATH=/path/to/bcc/lib/pkgconfig cmake ...`
+IF (CMAKE_SYSTEM_NAME STREQUAL "Linux" AND PKG_CONFIG_FOUND)
+    PKG_CHECK_MODULES(LIBBCC libbcc)
+
+    EXECUTE_PROCESS(
+        COMMAND uname -r
+        OUTPUT_VARIABLE KERNEL_VERSION
+        OUTPUT_STRIP_TRAILING_WHITESPACE
+    )
+
+    SET(REQUIRED_KERNEL_VERSION "4.9.0")
+    IF (KERNEL_VERSION VERSION_GREATER REQUIRED_KERNEL_VERSION OR KERNEL_VERSION VERSION_EQUAL REQUIRED_KERNEL_VERSION)
+        SET(HAVE_BPF_SUPPORTED_KERNEL ON)
+        MESSAGE(STATUS "Checking Linux kernel version ${KERNEL_VERSION} >= ${REQUIRED_KERNEL_VERSION} for h2olog - OK")
+    ENDIF ()
+
+    IF (WITH_DTRACE AND LIBBCC_FOUND AND HAVE_BPF_SUPPORTED_KERNEL)
+        SET(WITH_H2OLOG_DEFAULT ON)
+    ENDIF ()
+ENDIF ()
+
+OPTION(WITH_H2OLOG "use h2olog command" ${WITH_H2OLOG_DEFAULT})
+IF (WITH_H2OLOG)
+    MESSAGE(STATUS "Supporting h2olog")
+
+    SET(H2OLOG_SOURCE_FILES
+        "h2olog/h2olog.h"
+        "h2olog/h2olog.cc"
+        "h2olog/http.cc"
+        "h2olog/json.h"
+        "h2olog/json.cc"
+        "h2olog/quic.h"
+        "include/h2o/version.h"
+    )
+
+    FILE(COPY "deps/quicly/quicly-probes.d" DESTINATION "${CMAKE_CURRENT_BINARY_DIR}")
```

Comment:
Do we need to copy these .d files?

Reply:
This comes from a historical reason that `toru/h2olog` needs to copy .d files from `h2o/h2o`. 

Now no need to do so: 5b74935c6b9c4ad144d014cb8cde2b3ef713e825